TetrisStyleGame/Board.py

```diff
@@ -28,7 +28,7 @@ def _find_full_rows(self):
         for r, row in enumerate(self.rows):
             count = 0
             for cell in row:
-                if cell.color != EMPTY:
+                if cell.content != cell.EMPTY:
```

Comment:
The name change makes this read much better.

Maybe the board shouldn't be accessing the internal details of the cell though. What about adding a cell method `is_empty()`?